backend/tracim_backend/applications/share/lib.py

```diff
@@ -95,8 +95,9 @@ def share_content(
             except SMTPRecipientsRefused:
                 logger.warning(
                     self,
-                    "Share initied by {email} but SMTP server refuse to send notification".format(
-                        email=self._user.email
+                    "Share initied by user {} but "
+                    "SMTP server refuse to send notification ".format(
+                        self._user.email or self._user.username
```

Comment:
I suggest this `or` expression be but in a dedicated method. It is bound to be reused elsewhere, and it actually is in this PR later (I will put a comment there)

Reply:
changed in https://github.com/tracim/tracim/pull/2961/commits/2f620ef407bfd20a104c4a41e6e3669fab89938b